clang/test/Driver/sycl-mlir.cpp

```diff
@@ -251,6 +251,15 @@
 // CHK-INVOKE-ARG-PASS-RAISE: "{{.*}}mlir-translate" "-o" "{{.*}}.mlir" "--import-llvm" "{{.*}}.bc"
 // CHK-INVOKE-ARG-PASS-RAISE:  "{{.*}}cgeist" "-emit-llvm" "-S" "{{.*}}.cpp" "-sycl-use-host-module" "{{.*}}.mlir"
 
+// RUN: touch %t.o
```

Comment:
Would `touch` work when running the test on Windows?

Reply:
I know some utils are provided by lit to emulate missing tools. There is plenty of test doing `touch` without excluding windows, so I would say yes.